### DISCUSSION TITLE
nlmixr and nlmixr2

### DISCUSSION THREAD

**Discussion by Chickenlover0908** [nlmixrdevelopment/nlmixr#614] [General]
Hello, I am a student from china, I have read your paper about use 'nlmixr' package in popPK, It helped me a lot. My computer is macbook book M1, in the beginning I install 'nlmixr2‘ package，although I also install xcode and gfotran, there still some questions that I can not slove， so I install intel R and 'nlmixr',  then the fit run very well. Whether there are some difference between 'nlmixr' and 'nlmixr2' in model fit ?
thank you very much.

**Comment by mattfidler** [comment]
The changes are listed here

https://nlmixr2.github.io/nlmixr2est/news/index.html

**Reply by Chickenlover0908** [reply]
Thank you for your answer, thank you very much !!
